scripts/lib/api/conversionPipeline.ts

```diff
@@ -43,6 +43,33 @@ export async function runConversionPipeline(
   markdownPath: string,
   pkg: Pkg,
 ) {
+  const [files, maybeObjectsInv] = await determineFilePaths(htmlPath, pkg);
+  let initialResults = await convertFilesToMarkdown(
+    pkg,
+    htmlPath,
+    markdownPath,
+    files,
+  );
+
+  await createNecessaryDirs(initialResults);
+  await copyImages(pkg, htmlPath, initialResults);
```

Comment:
The copy used to happen at the very end. But the copy doesn't depend on anything else, so I moved it higher.

I think technically `copyImages` needs to happen before `postProcessResults` because `flattenFolders` will result in losing image paths. I don't like that dependency; it's not clear. I plan to fix this in a follow up by changing `flattenFolders` to correctly preserve image metadata.